### DISCUSSION TITLE
Connecting Miro with Github as a project management tool for collaboration

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66841] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Hey, has anyone had any luck in connecting Miro with Github, either via a workaround or existing apps out there? Similar to what already exists for Jira and Miro - but we have started using Github as a project tool too. 

Thanks!

**Comment by droumis** [comment]
I'm looking for something like this as well. There is a guide and video from Miro, but I haven't tried it yet and does not look like a simple process. As of writing, I think it also only works with the 'classic' GitHub projects, not the more recent version.

https://www.youtube.com/watch?v=ed_7TzPO0d4

https://developers.miro.com/docs/enable-2-way-sync-between-app-cards-and-github-cards

**Reply by ghost** [reply]
Hi @droumis, thanks for sharing what you've found! We have actually started building a public version ourselves to answer this pain point over the last couple of months that's in testing at the moment. If you're still interested I'd love to learn more about your use case and maybe get your thoughts on our version. You can email me at michelle.xie@servicerocket.com or sign up to our beta program here!

https://www.servicerocket.com/apps/co-innovator-program#join-us

**Comment by aidinsr** [comment]
Hey @droumis 

I’m Aidin from **ServiceRocket**, and we’re excited to team up with Miro for something cool. We've created a new integration that brings GitHub into Miro, designed to boost transparency and efficiency for product-engineering and design teams.

We're almost at the last stretch before launch of the product and would love for some Miro teams to help us polish this tool.Here are some sneak peeks: 
<img width="1789" alt="Image 4" src="https://github.com/community/community/assets/157421272/1c50fe26-c0d2-44ee-9005-3493b5efa320">
<img width="1789" alt="Image 3" src="https://github.com/community/community/assets/157421272/28d093f3-e263-449f-958a-7832f4d3ff8c">
<img width="1789" alt="Image 2" src="https://github.com/community/community/assets/157421272/7e3c6f9e-adef-415b-b6ff-d5fabd6ceb0b">

**What’s in it for you?**

By joining, you’ll get **early access** to our **GitHub for Miro** integration, where you can visualize engineering tasks and manage team activities like sprint reviews and roadmaps, all in Miro. Plus, you'll help shape the product and make your workflow smoother than ever!

Interested? Just reply here or send me an email at **aidin[at]servicerocket.com**.

**Reply by droumis** [reply]
Github projects at the org level can contain tasks from multiple repositories, even from other GitHub orgs. Can I import an org-level GitHub project board into Miro with your tool?

**Reply by jonathanappel** [reply]
> Github projects at the org level can contain tasks from multiple repositories, even from other GitHub orgs. Can I import an org-level GitHub project board into Miro with your tool?

At this stage you can only import issues as app cards, there is no board element for projects specifically. But yes, you can import issues from multiple GitHub orgs on the same board.

**Reply by PelinSmines** [reply]
@droumis @jonathanappel I am trying to connect my board to GitHub but I keep receiving an error message. I am attaching the ss for you to understand easier.
I click on Login with Github and then I receive this error message: https://github-miro.servicerocket.io/error?msg=Could%20not%20find%20GitHub%20state%20cookie.%20Restart%20the%20app%20and%20try%20again.

Under my GitHub account I see Miro among the Authorised GitHub apps via servicerocket. 

<img width="427" alt="Screenshot 2024-07-16 at 14 48 55" src="https://github.com/user-attachments/assets/15fafff7-f9ce-4d8f-9873-18c8600d18a7">



<img width="597" alt="Screenshot 2024-07-16 at 13 42 57" src="https://github.com/user-attachments/assets/4444729a-0611-4c9a-b6a9-ce170558491a">

**Reply by aidinsr** [reply]
Hey @PelinSmines can you please reach out to our support team [here](https://servicerocket-apps.atlassian.net/servicedesk/customer/portal/1/group/15) so we can help and troubleshoot this issue with you?

**Reply by PelinSmines** [reply]
Thanks @aidinsr I submitted the issue.
